Libraries/StyleSheet/StyleSheet.js

```diff
@@ -20,8 +20,8 @@ const flatten = require('flattenStyle');
 
 export type Styles = {[key: string]: Object};
 export type StyleSheet<S: Styles> = {[key: $Keys<S>]: number};
-export type StyleValue = {[key: string]: Object} | number | false | null;
-export type StyleProp = StyleValue | Array<StyleValue>;
+export type StyleValue = Object | number | false | null | void;
```

Comment:
According to `flattenStyle` any "falsy" value works so the users could do `thing && styles.blah`. I'm not sure if Flow has a special type representing falsy values, but might be a good idea to also include `''` (empty string) here

Reply:
Good spot, thanks.  I don't know of any special falsey type in Flow, so have just gone with empty string.